.github/workflows/test-docs-wiki-build.yml

```diff
@@ -0,0 +1,78 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+      - '298-begin-wiki-skeleton'
```

Comment:
Please @Schereo remove this line because it was used only to get documentation earlier